dm/openapi/swaggerui.go

```diff
@@ -111,16 +111,3 @@ func GetSwaggerHTML(config *SwaggerConfig) (html string, err error) {
 	}
 	return buf.String(), nil
 }
-
-// GetSwaggerJSON returns the swagger json.
-func GetSwaggerJSON() ([]byte, error) {
```

Comment:
`GetSwaggerJSON` seems to have no impact，but why was it deleted.

Reply:
GetSwaggerJSON is uesd with `echo` when openapi was introduced, as we use `gin` now, this func is no needed